### ISSUE TITLE
Migration from Onenote

### ISSUE THREAD

**Issue by nixsee** [foambubble/foam#151]
I was asked to open an issue here by @jsjoeio regarding making a recipe for migrating from Onenote to markdown/Foam using this tool that I've made. https://github.com/nixsee/ConvertOneNote2MarkDown

Not sure what is involved in that, but I'm happy to help. Or just take it and use as you please!

**Comment by jsjoeio** [comment]
Thanks for getting this started @nixsee! So from what I can guess, you've gone through the process of migrating markdown notes that lived in OneNote to markdown notes that live in Foam.

The goal of this issue is to the process that you went through and document it so that others can follow your the same with ease.

There are two steps I see here:
- [ ] add "Migrating from OneNote (stub)" to the [roadmap](https://foambubble.github.io/foam/roadmap) and link to this issue (following the same format you see for the other migration items)
- [ ] writing the migration guide itself and adding to the docs

For now, let's do 1. Once that's done, we'll focus on 2. How does that sound?

**Comment by nixsee** [comment]
I believe I got #1 finished by forking the roadmap and submitting a pull request?

**Comment by jsjoeio** [comment]
Yes, exactly! You're quick!

**Comment by jsjoeio** [comment]
Great! Once #152 lands, we can do 2. In the meantime, feel free to write up a markdown doc with the process and post it here if you want feedback. 

Otherwise, here's what I'm thinking:
- add Migration section to the [Foam Recipes](https://foambubble.github.io/foam/recipes)
- make it similar to the [Publish](https://foambubble.github.io/foam/recipes#publish), a bulleted list
- add a "Migrating from OneNote" bubble (it's own page) and link to it from the Foam Recipes page

Thoughts?

**Comment by dshemetov** [comment]
Hey, I'd like to help out. Can I give doing this docs work a shot?

**Comment by nixsee** [comment]
@dshemetov Sure! The instructions on the link to the Github repo for my OneNote script are fairly thorough, so you could probably  just copy and paste them and make some edits from there if you think its necessary. I can give it a final review to make sure it matches how the script works.

**Comment by dshemetov** [comment]
Thanks! Ok, I mostly just ported the Usage instructions in the README.md at your script repo. #200 is my fork.

**Comment by riccardoferretti** [comment]
Due to the high volume of issues, we're closing out older ones without recent activity. Please open a new issue if you need help!
